src/main/java/com/appium/utils/CapabilityManager.java

```diff
@@ -98,6 +98,19 @@ public String getAppiumServerPath(String host) throws Exception {
         return null;
     }
 
+    public String getAppiumServerPort(String host) throws Exception {
```

Comment:
Code repetition.. Same code just present in getAppiumServerPath. Extract function